cli/src/main.rs

```diff
@@ -192,10 +192,68 @@ fn run() -> Result<(), CliError> {
                                 .help("Path to private key file"),
                         )
                         .arg(
-                            Arg::with_name("path")
+                            Arg::with_name("node")
+                                .long("node")
                                 .takes_value(true)
                                 .required(true)
-                                .help("Path to a yaml file that defines the circuit proposal"),
+                                .multiple(true)
+                                .help(
+                                    "Node that are part of the circuit. \
+                                  Format: <node_id>::<endpoint>. \
+                                  Endpoint is optional is node alias has been set.",
```

Comment:
Endpoint is optional is node alias has been set -> Endpoint is optional if node alias has been set

Reply:
updated